### PR TITLE
Added issue templates

### PR DESCRIPTION
**Pull request by kirill-ivlev**
**Description** 
This PR introduces templates for customer issues,
You can check it out on the temporary demo repo [here](https://github.com/kirill-ivlev/issue-templates-test-tasks/issues/new/choose)

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/feature.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    attributes:
+      label: Task name
+  - type: textarea
+    id: what-happened
+    attributes:
+      label: Describe your feature request here
+      placeholder: Put your question here.
+      description: Please try to provide as much details as possible
+    validations:
+      required: true
```
Comment (vmapetr) [inline .github/ISSUE_TEMPLATE/feature.yml:21]: nit: newline here and in the config.yml?

**Review by vmapetr — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/config.yml [hunk 2]
```diff
@@ -0,0 +1 @@
+blank_issues_enabled: false
```
Comment (KonstantinTyukalov) [inline .github/ISSUE_TEMPLATE/config.yml:1]: We may place here links the same as here - https://github.com/microsoft/azure-pipelines-agent/blob/1096d5d8fff01a7ac571d4e91f339b3a1a3f63ec/.github/ISSUE_TEMPLATE/config.yml

File: .github/ISSUE_TEMPLATE/config.yml [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+contact_links:
+  - name: Developer Community
+    url: https://developercommunity.visualstudio.com/AzureDevOps
+    about: Feature ideas and problem reports go here.
```
Comment (KonstantinTyukalov) [inline .github/ISSUE_TEMPLATE/config.yml:5]: Not clear, which exactly issues and ideas?

File: .github/ISSUE_TEMPLATE/config.yml [hunk 4]
```diff
@@ -0,0 +1,15 @@
+blank_issues_enabled: false
+contact_links:
+  - name: Developer Community
+    url: https://developercommunity.visualstudio.com/AzureDevOps
+    about: For other Azure DevOps issuesЫ
```
Comment (KonstantinTyukalov) [inline .github/ISSUE_TEMPLATE/config.yml:5]:
```suggestion
    about: For other Azure DevOps issues
```

**Review by tkasparek — APPROVED** [review]

**Pull request merged**